### PR TITLE
Update pose estimator

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

File: src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java [hunk 1]
```diff
@@ -24,10 +24,10 @@ public class MoveToPos extends SwerveCommand {
     public static final double translationKp = 2;
     public static final double translationKi = 0;
     public static final double translationKd = 0;
-    
+
     private double tolerance = 0.05;
     private double rotationTolerance = 4;
-    private boolean debugDash = false;
+    private boolean debugDash = true;
```
Comment (roygold8) [inline src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java:30]: Probably this and all other debug functions should be removed or be in the dashboard subsystem (I think its better to remove)

**Review by roygold8 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java [hunk 2]
```diff
@@ -24,10 +24,10 @@ public class MoveToPos extends SwerveCommand {
     public static final double translationKp = 2;
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java:24]: finals are UPPER_SNAKE_CASE

File: src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java [hunk 3]
```diff
@@ -24,10 +24,10 @@ public class MoveToPos extends SwerveCommand {
...
     public static final double translationKi = 0;
     public static final double translationKd = 0;
-    
+
     private double tolerance = 0.05;
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java:28]: finals are UPPER_SNAKE_CASE

File: src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java [hunk 4]
```diff
@@ -49,36 +49,40 @@ public void initialize() {
...
+        yCalc = yController.atGoal() ? 0 : yCalc;
+        double rotationCalc = rotationController.calculate(SwerveChassis.getInstance().getChassisAngle());
+        rotationCalc = rotationController.atGoal() ? 0 : rotationCalc;
         swerve.moveByChassisSpeeds(
                 xCalc,
                 yCalc,
                 rotationCalc,
                 SwerveChassis.getInstance().getChassisAngle()
         );
         if(debugDash) { this.debugDashboard(xCalc, yCalc, rotationCalc); }
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java:79]: If you use one liners don't use {}, but even better, don't do it as a one liner.

File: src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java [hunk 5]
```diff
@@ -49,36 +49,40 @@ public void initialize() {
...
-        
+
     }
 
     @Override
     public boolean isFinished() {
         return super.isFinished() || (xController.atGoal() && yController.atGoal() && rotationController.atGoal());
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java:85]: Raz - I'm not sure I'm correct, but do we need to call super.isFinished();

**Pull request merged**